app/assets/stylesheets/spotlight/_featured_browse_categories_block.scss

```diff
@@ -0,0 +1,90 @@
+$featured-browse-category-border-color: #b3b3b3;
```

Comment:
Do we need our own special grays, or can we use bootstrap's (at least out-of-the-box)? These seem close-ish:

$legend-border-color c5c5c5
$gray-lighter fff